mojo/stdlib/stdlib/testing/suite.mojo

```diff
@@ -151,9 +151,7 @@ struct TestReport(Copyable, Movable, Writable):
         }
```

Comment:
This function (also `failed` and `skipped`) should take a `var String` and then move the string to avoid unnecessary copies.

```mojo
return {
    name = name^,
    duration_ns = duration_ns,
    result = TestResult.PASS,
}
```